python/ql/lib/semmle/python/dataflow/new/internal/DataFlowDispatchPointsTo.qll

```diff
@@ -497,59 +504,69 @@ class NonSpecialCall extends DataFlowSourceCall, TNonSpecialCall {
   override DataFlowCallable getEnclosingCallable() { result.getScope() = call.getNode().getScope() }
 }
 
+abstract class NonLibraryDataFlowSourceCall extends NonSpecialCall {
```

Comment:
I think this name needs a bit of workshopping. Could we come up with a better description of "non-library"?